packages/commercetools/composables/src/getters/storeGetters.ts

```diff
@@ -0,0 +1,172 @@
+import { UseStoreGetters } from '@vue-storefront/core';
+import { StoreQueryResult, Store, Channel, Address, Geometry, Maybe } from '../types/GraphQL';
+
+/**
+ * Types
+ */
+
+interface Predicate <T> {
+  (param: T): boolean;
+}
+
+interface Callable {
+  (...xs: any): any;
+}
+
+interface Localized {
+  locale: string;
+}
+
+type FilterCriteriaRecord <T extends Record<string, any>> = {
+  [K in keyof T]?: T[K] | Predicate<T[K]>;
+}
+
+type FilterCriteriaRecordValue <T extends Record<string, any>> =
+  FilterCriteriaRecord<T>[keyof T]
+
+type FilterCriteriaRecordField <T extends Record<string, any>> = [
+  keyof T, FilterCriteriaRecordValue<T>
+];
+
+interface AgnosticStore {
+  _storeID: string;
+  _channelID: string;
+  name: string;
+  id: string;
+  description: string;
+  locales: string[];
+  address: Maybe<Address>;
+  geoLocation: Maybe<Geometry>;
```

Comment:
both address and geoLocation need to be defined as Agnostic ones as well. Agnostic field means our format that represents something across all platforms, so all of data format will be transformed to that. The store, along with its data can be implemented in agnostic way as it's not really complex and can be easily mapped

Reply:
please consider Address as something simple, don't implement any complex interface that handles properties like streetName etc. perhaps addresLine1 and addressLine2 will be fine? and you can add private (_address) field that keeps original format (for customizations)